client/src/locales/en.json

```diff
@@ -66,8 +66,26 @@
   "about.intro-subheading": "{appName} helps you track all the office inventory you need!",
   "authorization.auth-box.description": "The best app to track your office inventory.",
   "authorization.auth-box.sign-in": "Sign in with:",
+  "authorization.auth-box.sign-in-google": "Sign in with Google",
   "authorization.auth-box.cta": "Try it out:",
   "authorization.auth-box.demo-button": "Live demo",
+  "authorization.actions.logout-fail": "Logout failed.",
+  "authorization.actions.login": "Login success.",
+  "authorization.auth-box.sign-up": "Sign up to app:",
+  "authorization.create-account": "Create account",
+  "authorization.sign-up": "Sign up",
+  "authorization.sign-up.confirmation-link-sent": "In order to complete sign up process please click the link that we have just sent to {data}.",
+  "authorization.input.username.label": "Name",
+  "authorization.input.username.invalid": "Please use max 32 chars.",
+  "authorization.input.username.empty": "Please enter your name.",
+  "authorization.input.email.label": "Email",
+  "authorization.input.email.invalid": "Please enter a valid email address.",
+  "authorization.input.password.label": "Password",
+  "authorization.input.password.confirm": "Confirm password",
+  "authorization.input.password.invalid": "Please use min 4 and max 32 signs and don't use space characters.",
+  "authorization.input.password.not-match": "Passwords do not match.",
+  "authorization.actions.sign-up.user-already-exist": "User of email {data} already exist.",
+  "authorization.actions.sign-up.failed": "Something went wrong while registration.",
```

Comment:
Should be `during` not `while` but why such information? Just make an general error and use it: `Something went wrong."